source/r_alias.c

```diff
@@ -533,7 +533,7 @@ void R_SetupAliasFrame (aliashdr_t *paliashdr, int frame, lerpdata_t *lerpdata)
 
 	if ((frame >= paliashdr->numframes) || (frame < 0))
 	{
-		Con_DPrintf ("R_AliasSetupFrame: no such frame %d for '%s'\n", frame, e->model->name);
```

Comment:
Why was this removed? What was being complained about? It should probably be properly fixed

Reply:
`"R_AliasSetupFrame: no such frame 9 for 'models/props/rebar.mdl'"`
directly after this print the frame is set to 0.

Reply:
also, `"R_AliasSetupFrame: no such frame 2 for 'models/props/teddy.mdl"` on Kino

Reply:
Thanks. We should be actually fixing these, not just suppressing the warning. Can you redo the commit without this change and open separate PRs?

Rebar is going to be nacht specific, teddy is because of a regression made by @blubs  that broke the sitting frame